content/24.vaccines-novel.md

```diff
@@ -431,59 +437,69 @@ Similar to BNT162b2, the mRNA-1273 vaccine was associated with mild-to-moderate
+In July of that year, as the Delta variant became dominant in the U.S.A., these estimates all fell, to an effectiveness of 76% for Moderna and 42% for Pfizer and effectiveness against hospitalization of 81% and 75%, respectively [@doi:10.1101/2021.08.06.21261707].
+
+With the emergence of the Omicron VOC, vaccine effectiveness has likely further declined.
+A study in a diverse cohort in Southern California, U.S.A. found the effectiveness of the Moderna vaccine in participants who had received only the primary course to be 44% [@doi:10.1038/s41591-022-01753-y].
+A study in South Africa compared case and hospitalization records from a 4-week period where Omicron was dominant to a 2-month period where Delta was dominant and found that the effectiveness against hospitalization during the Omicron wave was approximately 70% compared to 93% during the Delta wave [@doi:10.1056/NEJMc2119270].
+Similarly, a large study in England of 2.5 million individuals suggested that not only the variants circulating, but also the time since vaccination, played a large role in vaccine effectiveness [@doi:10.1056/NEJMoa2119451].
+Shortly after the BNT162b2 primary course, effectiveness against the Omicron VOC was as high as 65.5%, but this declined to below 10% by six months after the second dose.
+For mRNA-1273, the decline was from 75.1% to 14.9%.
+Therefore, it is unsurprising that in spite of vaccination programs, infection rates and hospitalization rates climbed in early 2022 in many Western countries including the United States [@url:https://www.nytimes.com/interactive/2021/us/covid-cases.html; @{url:https://www.washingtonpost.com/health/interactive/2022/omicron-comparison-cases-deaths-hospitalizations/}].
+
+On the side of safety, the only major concern that has been raised is a possible link between mRNA vaccination and myocarditis, especially in young men [@doi:10.15585/mmwr.mm7032e4].
+This concern began with case reports of a small number of cases of myocarditis following vaccination in several countries [@doi:10.1016/j.vaccine.2021.05.087; @doi:10.1001/jamacardio.2021.2828].
+Following these reports, the Israeli Ministry of Health began surveillance to monitor the occurrence of myocarditis [@doi:10.1056/NEJMoa2109730].
+They identified 283 cases, almost exactly half of which occurred following vaccination with Pfizer's BNT162b2.
+Close analysis of these cases determined that the vaccine did have a significant effect on the incidence of myocarditis; however, the rate of myocarditis remained low overall [@doi:10.1056/NEJMoa2109730].
+The identification of young men as a population at particular risk of this AE was supported, and the risk was found to be greater after the second dose than the first.
```

Comment:
Should we mention recovery? My understanding is that yes there is a prevalence but mortality is very low.

Reply:
```suggestion
The identification of young men as a population at particular risk of this AE was supported, and the risk was found to be greater after the second dose than the first.
However, most findings suggest that this AE does not have long-term negative effects; a 2021 meta-analysis identified 69 cases, all of which resulted in full recovery [@doi:10.7759/cureus.19240].
```

Reply:
Perfect, thanks @rando2